cpp/src/Slice/Parser.h

```diff
@@ -375,8 +375,10 @@ namespace Slice
         int line() const;
 
         std::string comment() const;
-        CommentPtr parseComment(bool stripMarkup) const;
-        CommentPtr parseComment(const std::string& text, bool stripMarkup) const;
```

Comment:
We only need one function, the 2nd one was a hack-around to let `ice2slice` massage it's links before parsing. The new `linkFormatter` approach solves this problem though.